lib/supavisor/helpers.ex

```diff
@@ -329,12 +329,12 @@ defmodule Supavisor.Helpers do
   def rpc(node, module, function, args, timeout \\ 15_000) do
     try do
       :erpc.call(node, module, function, args, timeout)
-      |> case do
-        {:EXIT, _} = badrpc -> {:error, {:badrpc, badrpc}}
-        result -> result
-      end
     catch
-      kind, reason -> {:error, {:badrpc, {kind, reason}}}
+      kind, reason ->
+        {:error, {:badrpc, {kind, reason}}}
```

Comment:
```suggestion
      kind, reason -> {:error, {:badrpc, {kind, reason}}}
```
```suggestion
      kind, reason ->
        {:error, {:badrpc, {kind, reason}}}
```

Reply:
I do not get this suggestions. You mean that it would be better to keep it in single line?

Reply:
yeah

Reply:
wdyt?

Reply:
Can be, other entries are also in single line, so it will be more consistent. Will push updated commit in a moment.